recipes/powershell2.rb

```diff
@@ -61,11 +63,19 @@
 
     include_recipe 'ms_dotnet2'
 
+    powershell_version_cmd = 'powershell.exe $PSVersionTable.PSVersion.Major'
+    shell_out = Mixlib::ShellOut.new(powershell_version_cmd)
+    shell_out.run_command
+    powershell_version = shell_out.stdout.to_f
+
+    # Reboot if user specifies immediate_reboot
```

Comment:
So to be clear, we always reboot, no matter what? If that's the case, we need to update the README.md to make that clear.

Separate thing in the `windows_reboot` recipe -- we have a guard expression that is always `false` -- if that's the case, we should remove the guard altogether.